spec/models/course/update_courses_gcse_requirements_spec.rb

```diff
@@ -0,0 +1,49 @@
+RSpec.describe Course, type: :model do
+  describe '#update_courses_gcse_requirements' do
+    let(:course) { create(:course, english: 1, maths: 1, science: 1) }
+
+    subject { course }
+
+    before do
+      course.update_courses_gcse_requirements(params)
+    end
+
+    context 'with a course with updated gcse requirements' do
+      context 'when all requirements are updated' do
+        let(:params) {
+          { "english" => "expect_to_achieve_before_training_begins",
+                        "maths" => "expect_to_achieve_before_training_begins",
+                        "science" => "expect_to_achieve_before_training_begins" }
+        }
+
+        its(:english_before_type_cast) { should eq 2 }
```

Comment:
what's the thinking behind using the non-enum values here? IMHO it'd be easier to check that the test is doing the right thing using the enums...

Reply:
Have made it more explicit in the req spec